src/__test__/components/ContentWrapper.test.jsx

```diff
@@ -104,6 +109,14 @@ describe('ContentWrapper', () => {
 
     store = makeStore();
 
+    navigator.userAgent = chromeUA;
```

Comment:
if you already do `Object.defineProperty(` in line 54 and you set chromeUA there, why do you have to set it again in here as user agent?